packages/trader/src/AppV2/Utils/trade-params-utils.tsx

```diff
@@ -42,3 +42,26 @@ export const isDigitContractWinning = (
     if (!contract_type || !win_conditions[contract_type]) return false;
     return win_conditions[contract_type];
 };
+
+export const focusAndOpenKeyboard = (element: HTMLInputElement | null) => {
```

Comment:
IOS prevent input from focusing with showing keyboard before user clicks on it. The only solution that helped me was taken from [here](https://stackoverflow.com/questions/54424729/ios-show-keyboard-on-input-focus).
Before, I tried with autofocus attribute, also called focus() and click() as it is and inside of timer. Nothing helps and autofocus is even not supported in IOS.
IOS prevent input from focusing with showing keyboard before user clicks on it. The only solution that helped me was taken from [here](https://stackoverflow.com/questions/54424729/ios-show-keyboard-on-input-focus).
Before, I tried with autofocus attribute, also called focus() and click() as it is and inside of timer. Nothing helps and autofocus is even not supported in IOS.
<img width="1361" alt="Screenshot 2024-07-29 at 6 44 50 PM" src="https://github.com/user-attachments/assets/787a5b01-a940-47d1-935f-98c59cf631f5">

Reply:
@kate-deriv all these Vanilla JS methods - `createElement`, `appendChild` and `removeChild` - working directly with the DOM can be easily replaced in React fashion.
What you have to do:
1. add a conditionally rendered `<input />` styled in a way you wrote all the styles below using a classname
2. the temporary input should appear next to the `TextFieldWithSteppers` when you call `focusAndOpenKeyboard()`.
3. since the position of the temporary input depends on the offset dimensions of TextFieldWithSteppers, you can put a ref on it but use it only for calculated styles that cannot be added via CSS classname.
;)